datalad/support/gitrepo.py

```diff
@@ -1116,6 +1279,17 @@ def add_submodule(self, path, name=None, url=None, branch=None):
         cmd += [url, path]
         self._git_custom_command('', cmd)
 
+    def deinit_submodule(self, path, **kwargs):
+        """
+        Parameters
+        ----------
+        path
+        """
+
+        kwargs['insert_kwargs_after'] = 'deinit'
```

Comment:
to be safe, make a copy first: `kwargs = kwargs.copy()`, or use `kwargs = datalad.utils.updated(kwargs, {'insert_kwargs_after': 'deinit'})`